### PR TITLE
Box 2

### PR DESCRIPTION
**Pull request by Jiaqi-beep**
Description: boxplot with altered thickness and colors

- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/box_2.ncl
- Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/box_2_lg.png
- Issue #346

box2

### CODE REVIEW

File: Plots/Boxplots/NCL_box_2.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
+"""
+NCL_box_1.py
+===============
```
Comment (anissa111) [inline Plots/Boxplots/NCL_box_2.py:3]:
```suggestion
NCL_box_1.py
============
```

**Review by anissa111 — COMMENTED** [review]
This looks good so far! I think what you mentioned as looking a little funny is that the original script modifies the width of the boxplot, which I don't think you're recreating. Note that in the original, both the box itself and the top and bottom range markers are reduced in width.

Also, when you make a new branch, make sure you're branching off of main so that extra files don't get pulled in! It looks like you've got your xy_6 plot attached to this PR, too.

File: Plots/Boxplots/NCL_box_2.py (hunk 1 above)
Comment (Jiaqi-beep) [reply Plots/Boxplots/NCL_box_2.py:3]: I see. Does that mean I should be on the main branch when I do git checkout -b newbranch?

Comment (anissa111) [reply Plots/Boxplots/NCL_box_2.py]: Yup exactly that!

Comment (Jiaqi-beep) [reply Plots/Boxplots/NCL_box_2.py]: Got it. Should I try and delete those files from this branch? Also by length of the range markers, do you mean the length of the whisker? (I'm assuming using the whis argument)

Comment (anissa111) [reply Plots/Boxplots/NCL_box_2.py]: `whis` is not actually the argument you want. I think you'll actually want to modify `cap`

File: Plots/Boxplots/NCL_box_2.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
+"""
+NCL_box_1.py
```
Comment (anissa111) [inline Plots/Boxplots/NCL_box_2.py:2]:
```suggestion
NCL_box_2.py
```

**Review by michaelavs — APPROVED** [review]
Looks good to me!

**Review by anissa111 — APPROVED** [review]
Looks good to me, too!

**Pull request merged**